STYLE_GUIDE.md

```diff
@@ -0,0 +1,173 @@
+# Contributing
+
+## Branching Strategy
+
+Feature -> Develop* -> Master -> Staging -> Deployment
+* default branch
+
+- Deployment: actual production environment available publicly
+- Staging: simulated deployed state only available internally
+- Master: always "good" source of truth. Only certain members can merge into master
+- Develop: "experimental / latest and greatest" code and default branch for all members
+- Feature: individual feature implementations per-user
+
+## Internal Developer Workflow
+
+1. Clone the repo.
+2. Switch to the 'develop' branch.
+3. Create a new 'feature' branch named by feature (e.g. add-authentication).
```

Comment:
Do we want to follow the convention of naming the type of branch it is? e.g: `feat/search`, `bug/duplicate-users`, `refactor/search-filter`?

Reply:
Yeah, we should maintain the convention and keep it consistent. 

Do we prefer "feat" or "feature"?